### PR TITLE
Issue 1 

### PR DESCRIPTION
**Pull request by anandsimer**
pylint score: 8.0

### CODE REVIEW

File: server-side/server.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+"""This module responds to the REST invocations from client side app."""
```
Comment (hennasingh) [inline server-side/server.py]: Also add, for current project (Client is Kotlin Android app) A little more description will be helpful

File: server-side/server.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+from server_handler import ServerHandle
+
+app = Flask(__name__)
+
+server_prop = ServerHandle()
+
+
+@app.route('/', strict_slashes=False)
+def root():
+    """Root method which returns with Server hostname.
```
Comment (hennasingh) [inline server-side/server.py]: Please add what server is for this current project, a little more description

File: server-side/server_handler.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
+""" Getter and Setter class for fetching hostname and IP."""
+
+import socket
+import psutil
+
+
+class ServerHandle:
+    """class with getter methods returning various system variables."""
```
Comment (hennasingh) [inline server-side/server_handler.py]: detailed explanation please- which system variables

File: server-side/server.py (hunk 2 above)
Comment (anandsimer) [reply server-side/server.py]:
```suggestion
    """Root method which returns with Server hostname.
```

This is done, added a little more info to the DocString.

File: server-side/server_handler.py (hunk 3 above)
Comment (anandsimer) [reply server-side/server_handler.py]:
```suggestion
    """Root method which returns with Server hostname.
```

This is done, added a little more info to the DocString.

File: server-side/server.py (hunk 1 above)
Comment (anandsimer) [reply server-side/server.py]:
```suggestion
"""This module responds to the REST invocations from the client-side app."""
```
Added little more description.

**Comment by anandsimer** [comment]
Please review the changes have been made per review comments.

**Pull request merged**